pkg/featuregate/features.go

```diff
@@ -19,6 +19,10 @@ package featuregate
 import "fmt"
 
 const (
+	// ForcedAdoptResources is a feature gate for enabling forced adoption of resources
+	// by annotation
+	ForcedAdoptResources = "ForcedAdoptResources"
```

Comment:
It feels like "Forced" is a bit of a strong word, how about just `AdoptedResources`? the only thing is that it might bring confusion with the AdoptedResource CR